backend/model/top_container.rb

```diff
@@ -306,6 +306,45 @@ def self.bulk_update_container_profile(ids, container_profile_uri)
   end
 
 
+  def self.bulk_update_location(ids, location_uri)
+    out = {:records_updated => ids.length}
+
+    relationship = TopContainer.find_relationship(:top_container_housed_at)
+
+    begin
+      relationship.handle_delete(relationship.find_by_participant_ids(TopContainer, ids).keep_if{|v| v.status == 'current'}.map(&:id))
```

Comment:
Hi @payten - just about to push the change to select - thanks.

On the other point, they were adamant that they only want one location. Also, there's a validation rule that says only temporary locations can be previous ...